packages/core-components/src/layout/ErrorPage/ErrorPage.tsx

```diff
@@ -62,7 +63,7 @@ const useStyles = makeStyles<BackstageTheme>(
  *
  */
 export function ErrorPage(props: IErrorPageProps) {
-  const { status, statusMessage, additionalInfo } = props;
+  const { status, statusMessage, additionalInfo, supportUrl } = props;
```

Comment:
Could you explain the use case you're trying to solve here, I mean why is the URL in the configuration is not enough?

Reply:
The url in the config applies to the whole app, which is great if you have a centralised support team, but we need to add a specific support channel for a specific plugin so that the users get to our teams channel :/

Reply:
Got it, thanks for explaining, yes, good real use case!